tradeexecutor/visual/single_pair.py

```diff
@@ -70,7 +70,8 @@ def get_position_hover_text(p: TradingPosition) -> str:
         ]
 
     text += [
-        f"Entry price: {first_trade.planned_mid_price:.2f} USD (mid price)",
+        # TODO: not sure why this failed for short position
+        # f"Entry price: {first_trade.planned_mid_price:.2f} USD (mid price)",
```

Comment:
@miohtama I can look closer at this tomorrow, but do you have any idea why `planned_mid_price` is None here?